tests/Medidata.ZipkinTracer.Core.Test/ZipkinClientTests.cs

```diff
@@ -569,6 +569,48 @@ public void RecordBinary_IsTraceOnIsFalse_DoesNotAddBinaryAnnotation()
             Assert.AreEqual(0, testSpan.Binary_annotations.Count, "There are annotations but the trace is off.");
         }
 
+        [TestMethod]
+        [TestCategory("TraceRecordTests")]
+        public void RecordLocalComponent_WithNotNullValue_AddsLocalComponentAnnotation()
+        {
+            // Arrange
+            var testValue = "Some Value";
+            var tracerClient = SetupZipkinClient();
+            spanTracerStub = MockRepository.GenerateStub<SpanTracer>(spanCollectorStub, MockRepository.GenerateStub<ServiceEndpoint>(), new List<string>(), fixture.Create<string>(), fixture.Create<string>());
```

Comment:
80 chars is the old school, super conservative limit.
120 is the modern limit. One of the reasons it's not above is that in github the diff windows only display up to 121 chars so I would advise you against going above that.
Horizontal scrolling is annoying and having a wide monitor doesn't help here ;)